src/components/editors/SceneEditor.tsx

```diff
@@ -816,6 +853,25 @@ export const SceneEditor = ({ id }: SceneEditorProps) => {
                     name="playerSpriteSheetId"
                     label={l10n("FIELD_SCENE_SPRITE_PALETTES")}
                   >
+                  {monoEnabled &&
+                    <PaletteButtons>
```

Comment:
Perhaps `PaletteButtons` was the wrong wrapper node to use here, but I used it as a quick and dirty way of keeping the OBP0 and OBP1 palette pickers on the same row.